Samples/UnifiedAPI/ArmoniK.Samples.Client/appsettings.json

```diff
@@ -42,7 +42,7 @@
   },
 
   "Grpc": {
-    "EndPoint": "http://ANEO-SB2-8454-wsl.local:5001"
+    "EndPoint": "http://a810dc85181ab4ac682d984366e81554-1922853256.eu-west-3.elb.amazonaws.com:5001"
```

Comment:
Not a fan of having this in the default config, but I guess in this case it doesn't matter ?

Reply:
We nedd to ignore change on this file in gitignore ?